google/cloud/storage/tests/object_file_integration_test.cc

```diff
@@ -625,6 +625,51 @@ TEST_F(ObjectFileIntegrationTest, UploadPortionRegularFile) {
   EXPECT_EQ(0, std::remove(file_name.c_str()));
 }
 
+TEST_F(ObjectFileIntegrationTest, ResumableUploadFileCustomHeader) {
+  // Test relies on emulator for capturing custom header
+  if (!UsingEmulator()) GTEST_SKIP();
```

Comment:
@vnvo2409 FYI, the test is skipped if not using the emulator.

Reply:
Ah sorry again 😓 I am not at my best today.